.gitignore

```diff
@@ -7,6 +7,9 @@ pids
 *.pid
 *.seed
 
+# Configuration for integration tests
+test/config.json
```

Comment:
Per Jamie's (reasonable and well argumented) request to scrap integration tests, this is gone